frontend/src/pages/projects/notebook/utils.ts

```diff
@@ -252,3 +252,6 @@ export const useNotebookStatus = (
 
 export const getEventFullMessage = (event: EventKind): string =>
   `${getEventTimestamp(event)} [${event.type}] ${event.message}`;
+
+export const getNotebooksNames = (notebooks: NotebookKind[]): string[] =>
+  notebooks.map((notebook) => notebook.metadata.name);
```

Comment:
huh? Just do this where you need it? It's not custom logic, it's just working with k8s objects. If anything we can getResourceName in our base utils for projects -- but it seems kinda goofy